### DISCUSSION TITLE
Nuova versione 1.4.2

### DISCUSSION THREAD

**Discussion by trinko** [iisgiua/giuaschool#103] [Annunci]
La nuova versione ha una nuova procedura per l'installazione iniziale o l'aggiornamento di una versione esistente.
Dopo aver copiato i file nella cartella di destinazione, andare alla pagina seguente e seguire le istruzioni:

- **http://nome_sito/install/index.php**

Maggiori dettagli sulla release:
[Versione 1.4.2](https://github.com/trinko/giuaschool/releases/tag/v1.4.2)

**Comment by sempliciniluigi89** [comment]
La procedura è simile a quella di LAMPSchool giusto?

**Reply by trinko** [reply]
Si, dovrebbe essere simile
